dag/forests.yml

```diff
@@ -0,0 +1,9 @@
+steps:
+  # Intact Forest Landscapes
+  data://meadow/forests/2024-05-08/ifl:
```

Comment:
I think it would be more useful to spell out this acronym, since there are many IFL things, and this one is not particularly well known (at least to me). But if it's too much hassle, you can leave it as-is.